.github/workflows/release.yaml

```diff
@@ -388,192 +388,193 @@ jobs:
-#    runs-on: ubuntu-latest
-#    container: devkitpro/devkitarm
-#    steps:
-#      - name: Download Artifact with Configuration Information
-#        uses: actions/download-artifact@v4
-#        with:
-#          name: ${{ env.CONFIG_ARTIFACT }}
-
-#      - name: Extract Configuration Information and Store in Step Outputs
-#        id: store_config
-#        run: |
-#          name=`sed -E -n -e 's/name= //p' $CONFIG_ARTIFACT_PATH`
-#          echo "name=$name" >> $GITHUB_OUTPUT
-#          prog=`sed -E -n -e 's/prog= //p' $CONFIG_ARTIFACT_PATH`
-#          echo "prog=$prog" >> $GITHUB_OUTPUT
-#          version=`sed -E -n -e 's/version= //p' $CONFIG_ARTIFACT_PATH`
-#          echo "version=$version" >> $GITHUB_OUTPUT
-
-#      - name: Clone Project
-#        uses: actions/checkout@v4
-#        with:
-#          fetch-depth: 0
-
-#      - name: Download Artifact with Converted Documents
-#        uses: actions/download-artifact@v4
-#        with:
-#          name: ${{ env.DOC_ARTIFACT }}
-
-#      - name: Extract Converted Documents
-#        run: |
-#          tar -xf ${{ env.DOC_ARTIFACT_PATH }}
-
-#      - name: Create Nintendo DS Archive
-#        id: create_nintendo_ds_archive
-#        shell: bash
-#        run: |
-#          pushd src/
-#          make -f Makefile.nds
-#          popd
-#          test -e src/${{ steps.store_config.outputs.prog }}.nds
-
-#          archive_prefix=${{ steps.store_config.outputs.name }}-${{ steps.store_config.outputs.version }}
-#          echo "archive_file=${archive_prefix}-nds.zip" >> $GITHUB_OUTPUT
-#          mkdir -v ${{ steps.store_config.outputs.prog }}/
-#          cp -v src/${{ steps.store_config.outputs.prog }}.nds ${archive_prefix}.nds
-#          cp -rv lib ${{ steps.store_config.outputs.prog }}/
-#          zip -r ${archive_prefix}-nds.zip ${archive_prefix}.nds ${{ steps.store_config.outputs.prog }}/
-
-#      - name: Create Artifact with Nintendo DS Archive Path
-#        run: |
-#          echo archive_path= '${{ steps.create_nintendo_ds_archive.outputs.archive_file }}' > $NIN_DS_ARTIFACT_PATH
-
-#      - name: Upload Artifact with Nintendo DS Archive Path
-#        uses: actions/upload-artifact@v4
-#        with:
-#          name: ${{ env.NIN_DS_ARTIFACT }}
-#          path: ${{ env.NIN_DS_ARTIFACT_PATH }}
-#          retention-days: 1
-
-#      - name: Upload Nintendo DS Archive as Artifact
-#        uses: actions/upload-artifact@v4
-#        with:
-#          name: ${{ env.NIN_DS_ARCHIVE_ARTIFACT }}
-#          path: ${{ steps.create_nintendo_ds_archive.outputs.archive_file }}
-#          retention-days: 1
+      - name: Build Nintendo 3DS (cia)
+        id: create_nintendo_3ds_cia
+        shell: bash
+        run: |
+          pushd src/
+          prog='${{ steps.store_config.outputs.prog }}'
+          procname=`echo "$prog" | head -c 8`
+          test -e "$prog".3dsx
+          cxitool --name=angband --code=CTR-P-ANGBAND --tid=000400000616E600 angband.3dsx angband.cxi
```

Comment:
In my hands, writing lines 469 and 470 as

```
cxitool --name="$procname" --code=CTR-P-`echo "$procname" | tr [:lower:] [:upper:]` --tid=000400000616E600 "$prog".3dsx "$prog".cxi
makerom -v -f cia -o "$prog".cia -target t -i "$prog".cxi:0:0 -ignoresign -icon icon.smdh
```

ran.  I did not check if the generated files were usable.  What problem did you have when trying to use the variables rather than hardwired names?

It is desirable to avoid hardwiring "angband" or "ANGBAND" in the commands here so forks of Angband that reuse the workflows have less to change.

Reply:
In that failed run, the name of the variable assigned to at line 465 had been changed from prog to progname but line 476 referenced prog.  As the pull request stands, it is back to using prog so replacing lines 469 and 470 with

```
cxitool --name="$procname" --code=CTR-P-`echo "$procname" | tr [:lower:] [:upper:]` --tid=000400000616E600 "$prog".3dsx "$prog".cxi
makerom -v -f cia -o "$prog".cia -target t -i "$prog".cxi:0:0 -ignoresign -icon icon.smdh
```

avoids hardwiring the project's name in the workflow, and all a fork has to replace is the ID passed to --tid.